packages/hydra-cli/src/templates/graphql-server/src/WarthogBaseService.ts.mst

```diff
@@ -16,146 +16,30 @@ export class WarthogBaseService<E extends BaseModel> extends BaseService<E> {
-      where: WhereFilterAttributes
-    ): SelectQueryBuilder<E> => {
-      // where is of shape { userName_contains: 'a' }
-      Object.keys(where).forEach((k: string) => {
-        const paramKey = `${paramKeyPrefix}${paramKeyCounter.counter}`;
-        // increment counter each time we add a new where clause so that TypeORM doesn't reuse our input variables
-        paramKeyCounter.counter = paramKeyCounter.counter + 1;
-        const key = k as keyof W; // userName_contains
-        const parts = key.toString().split('_'); // ['userName', 'contains']
-        const attr = parts[0]; // userName
-        const operator = parts.length > 1 ? parts[1] : 'eq'; // contains
-
-        return addQueryBuilderWhereItem(
-          qb,
-          paramKey,
-          this.attrToDBColumn(attr),
-          operator,
-          where[key]
-        );
-      });
-      return qb;
-    };
-
-    // WhereExpression comes in the following shape:
-    // {
-    //   AND?: WhereInput[];
-    //   OR?: WhereInput[];
-    //   [key: string]: string | number | null;
-    // }
-    const processWhereInput = (
-      qb: SelectQueryBuilder<E>,
-      where: WhereExpression
-    ): SelectQueryBuilder<E> => {
-      const { AND, OR, ...rest } = where;
-
-      if (AND && AND.length) {
-        const ands = AND.filter((value) => JSON.stringify(value) !== '{}');
-        if (ands.length) {
-          qb.andWhere(
-            new Brackets((qb2) => {
-              ands.forEach((where: WhereExpression) => {
-                if (Object.keys(where).length === 0) {
-                  return; // disregard empty where objects
-                }
-                qb2.andWhere(
-                  new Brackets((qb3) => {
-                    processWhereInput(qb3 as SelectQueryBuilder<any>, where);
-                    return qb3;
-                  })
-                );
-              });
-            })
-          );
-        }
-      }
-
-      if (OR && OR.length) {
-        const ors = OR.filter((value) => JSON.stringify(value) !== '{}');
-        if (ors.length) {
-          qb.andWhere(
-            new Brackets((qb2) => {
-              ors.forEach((where: WhereExpression) => {
-                if (Object.keys(where).length === 0) {
-                  return; // disregard empty where objects
-                }
-
-                qb2.orWhere(
-                  new Brackets((qb3) => {
-                    processWhereInput(qb3 as SelectQueryBuilder<any>, where);
-                    return qb3;
-                  })
-                );
-              });
-            })
-          );
+      fields.forEach((field) => {
+        if (field === 'id' || !this.columnMap[field]) {
+          return;
         }
-      }
-
-      if (rest) {
-        processWheres(qb, rest);
-      }
-      return qb;
-    };
 
-    if (Object.keys(where).length) {
-      processWhereInput(qb, where);
+        qb = qb.addSelect(`${this.klass}.${field}`, aliases(field));
```

Comment:
Not sure if it's applicable, but please have a look at this commit: https://github.com/subsquid/hydra/commit/ca4bcd914926beceee8be79d376ce628f298ee7c

Reply:
This change in the referenced commit should be the same as in this PR, just formulated a bit differently.